### DISCUSSION TITLE
Is there any equivalent to createStandaloneToast in chakra ui 3.0?

### DISCUSSION THREAD

**Discussion by bsolarski** [chakra-ui/chakra-ui#9029] [Q&A]
I used to use the createStandaloneToast method, now I have a message in the console that such a method is not exported, and the migration description does not provide any instructions on how to replace it. I wanted to migrate to ChaktraUI 3.0 but I don't know how to deal with it.

**Comment by nikrokss** [comment]
Use this from the documentation instead of createStandaloneToast():

```
 toaster.create({
   title: 'Successfully created!',
   status: 'success',
 });
```
          
[Documentation](https://chakra-ui.com/docs/components/toast)
